source/aggregation-pipeline-builder.txt

```diff
@@ -44,173 +44,229 @@ Create an Aggregation Pipeline
 
 .. include:: /includes/steps/create-agg-pipeline.rst
 
+Specify Custom Collation
```

Comment:
I don't see this new procedure staged anywhere.

Reply:
It's a bit hidden. It's nested in the Create an Agg Pipeline Procedure. See: https://docs-mongodbcom-staging.corp.mongodb.com/compass/jeffreyallen/DOCSP-6082/aggregation-pipeline-builder.html#specify-custom-collation